x-pack/test/functional/apps/ml/anomaly_detection_jobs/job_list.ts

```diff
@@ -0,0 +1,146 @@
+      influencers: ['airline'],
+      detectors: [
+        { function: 'mean', field_name: 'responsetime', partition_field_name: 'airline' },
+      ],
+    },
+    data_description: { time_field: '@timestamp' },
+    analysis_limits: { model_memory_limit: '50mb' },
+    model_plot_config: { enabled: true },
+  };
+
+  describe('job list', function () {
+    this.tags(['ml']);
+
+    before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await ml.testResources.createDataViewIfNeeded('ft_farequote', '@timestamp');
+      await ml.testResources.setKibanaTimeZoneToUTC();
+
+      const JOB_CONFIG = ml.commonConfig.getADFqSingleMetricJobConfig(jobId);
+      const DATAFEED_CONFIG = ml.commonConfig.getADFqDatafeedConfig(jobId);
+
+      await ml.api.createAndRunAnomalyDetectionLookbackJob(JOB_CONFIG, DATAFEED_CONFIG);
+
+      await ml.api.waitForJobState(jobId, JOB_STATE.CLOSED);
+      await ml.api.openAnomalyDetectionJob(jobId);
+
+      const { body, status } = await supertest
+        .post(`/internal/ml/anomaly_detectors/${jobId}/_forecast`)
+        .auth(USER.ML_POWERUSER, ml.securityCommon.getPasswordForUser(USER.ML_POWERUSER))
+        .set(getCommonRequestHeader('1'))
+        .send({ duration: '1d' });
+      ml.api.assertResponseStatusCode(200, status, body);
+
+      await ml.api.assertForecastResultsExist(jobId);
+
+      await ml.api.createAnomalyDetectionJob(multiMetricJobConfig);
+
+      await ml.navigation.navigateToMl();
+      await ml.navigation.navigateToSettings();
+      await ml.settings.navigateToCalendarManagement();
+
+      await ml.settingsCalendar.assertCreateCalendarButtonEnabled(true);
+      await ml.settingsCalendar.navigateToCalendarCreationPage();
+
+      await ml.settingsCalendar.waitForFormEnabled();
+
+      await ml.settingsCalendar.assertJobSelectionExists();
+      await ml.settingsCalendar.assertJobSelectionEnabled(true);
+      await ml.settingsCalendar.assertJobGroupSelectionExists();
+      await ml.settingsCalendar.assertJobGroupSelectionEnabled(true);
+
+      await ml.settingsCalendar.setCalendarId(calendarId);
+
+      await ml.settingsCalendar.selectJob(jobId);
+
+      await ml.settingsCalendar.saveCalendar();
+      await ml.settingsCalendar.assertCalendarRowExists(calendarId);
+
+      await ml.securityUI.loginAsMlPowerUser();
+
+      await ml.navigation.navigateToMl();
+
+      await ml.navigation.navigateToJobManagement();
+    });
+
+    after(async () => {
+      await ml.api.closeAnomalyDetectionJob(jobId);
+      await ml.api.cleanMlIndices();
+      await ml.api.deleteAnomalyDetectionJobES(jobId);
+      await ml.testResources.deleteDataViewByTitle('ft_farequote');
+    });
+
+    it('expanded row with connected calendar shows link to calendar', async () => {
+      await ml.testExecution.logTestStep('calendar present for connected job');
+      await ml.expandedJobDetails.assertCalendarPresent(jobId, calendarId);
+    });
```

Comment:
If the whole test (`it`) is very short (like in this case just one step), we don't have to add a `logTestStep` as the test title already mentions what's being asserted. The `logTestStep` is mostly there to structure longer test flows to make it easier to find the step where things went wrong when reading through a failed test log.
Same for other tests in this file, I just won't repeat this comment again and again.